dnf-behave-tests/features/microdnf/protect-running-kernel.feature

```diff
@@ -0,0 +1,45 @@
+@no_installroot
+Feature: Protect running kernel
+
+
+Background: Install fake kernel
+  Given I use repository "repoquery-unsatisfied"
```

Comment:
The tests are actually using a repository belonging to another feature file, I think we should avoid this and likely create a dedicated repo for these tests? Or, we could create a "shared" repo named accordingly, e.g. `kernel-protected`. Although, the `repoquery-unsatisfied` actually tests quite a special case that's just related to a kernel package.

Reply:
I'll create a dedicated repository. I also just came up with one more test that will require additional package, so it will be better that way.